config.go

```diff
@@ -192,19 +188,95 @@ func loadConfiguration(configPath string) (oci.RuntimeConfig, ShimConfig, error)
 
 			config.ProxyType = vc.CCProxyType
 			config.ProxyConfig = pConfig
+
+			break
+		}
+	}
+
+	for k, agent := range tomlConf.Agent {
+		switch k {
+		case hyperstartAgent:
+			path := filepath.Join(agent.pauseBundlePath(),
+				pauseBinRelativePath)
```

Comment:
@sboeuf @jodh-intel this is hyperstart specific and we only need a statically linked pause binary to give to hyperstart so that it can create the pause container in the pod.
Thus an OCI config file is not needed and we can just glue the bundle path (bundle may be the wrong term here...) with the pause binary relative path.